src/js/components/Notification/stories/TitleAndMessage.js

```diff
@@ -18,6 +18,7 @@ const TitleAndMessageNotification = () => {
       {visible && (
         <Notification
           toast
+          position="top-right"
```

Comment:
instead of adjusting this story, can we add a jest test that catches the position in a snapshot?

Reply:
Yes good point ill adjust

Reply:
Jest test looks good -- can we remove from here now?

Reply:
yes I will update